qpid-docker/README.md

```diff
@@ -0,0 +1,198 @@
+
+Build the parent project using command
+
+```
+mvn clean install -DskipTests=true
+```
+
+Navigate to the module 'qpid-docker':
+
+```
+cd qpid-docker
+```
+
+Execute command
+
+```
+docker-build.sh --local-dist-path <PATH_TO_LOCAL_QPID_DISTRIBUTION>
+```
+
+This will copy all the files necessary to build the pre-configured Docker image and provide you with additional 
+instructions. Follow these instructions to finish building the image you want based on one of the provided Docker file 
+or even one of your own.
+
+If you would rather use an official Apache release in your image rather than a local release then run the following 
+command from the qpid-docker directory where <QPID_VERSION> is the release version you wish to use (e.g. 9.1.0):
+
+```
+docker-build.sh --qpid-version <QPID_VERSION>
+```
+
+This will download the Qpid Broker-J release and copy all the files necessary to build the pre-configured Docker image
+and provide you with additional instructions. Follow these instructions to finish building the image you want based on 
+the provided Docker file or even one of your own.
+
+### Container Structure
+
+Broker-J files are copied to the folder /qpid-broker-j \
+This folder belongs to user qpid, which is part of the root group. Java process is executed under the qpid user as well.
+
+### Running the Container
+
+Container can be started using following command:
+```
+docker run -d -p 5672:5672 -p 8080:8080 --name qpid <IMAGE_NAME>
+```
+There are two ports exposed: 5672 for AMQP connections and 8080 for HTTP connections.
+
+There are following environment variables available when running the container:
+
+| Environment Variable | Description                                                                  |
+|----------------------|------------------------------------------------------------------------------|
+| JAVA_GC              | JVM Garbage Collector parameters, default value "-XX:+UseG1GC"               |
+| JAVA_MEM             | JVM memory parameters, default value "-Xmx300m -XX:MaxDirectMemorySize=200m" |
+| JAVA_OPTS            | Further JVM parameters, empty by default                                     |
+
+#### Container Volume
+
+Before starting the container a volume should be created:
+
+```
+docker volume create --driver local --opt device=<PATH_TO_FOLDER_ON_HOST_MACHINE> --opt type=none --opt o=bind qpid_volume
+```
+
+Then container using created volume can be started as follows:
+
+```
+docker run -d -p 5672:5672 -p 8080:8080 -v qpid_volume:/qpid-broker-j/work --name qpid <IMAGE_NAME>
+```
+or
+```
+podman run -d -p 5672:5672 -p 8080:8080 -v qpid_volume:/qpid-broker-j/work:Z --name qpid <IMAGE_NAME>
+```
+When container is started for the first time, configuration files become available in the mapped folder being persisted 
+after subsequent restarts.
+
+### Broker Users
+
+Default configuration provides two preconfigured broker users:
+- admin (password 'admin')
+- guest (password 'guest')
```

Comment:
The hard coded user+pass could be a bit annoying. Could they be updateable via env variable and either passed as config properties when starting the broker, or manipulate the config file when copying it?

Does it really need 2 users by default if others can be added by management or custom config?